modules/qt/hm.nix

```
@@ -0,0 +1,63 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: {
+  options.stylix.targets.qt = {
+    enable = config.lib.stylix.mkEnableTarget "QT" pkgs.stdenv.hostPlatform.isLinux;
+    iconThemeName = lib.mkOption {
+      description = "Default QT Icons";
+      type = lib.types.str;
+      default = "ePapirus-Dark";
+    };
+  };
+
+  config = lib.mkIf config.stylix.targets.qt.enable (let
+    cfg = config.stylix.targets.qt;
+    kvconfig = config.lib.stylix.colors {
+      template = ./kvconfig.mustache;
+      extension = ".kvconfig";
+    };
+    svg = config.lib.stylix.colors {
+      template = ./kvantum-svg.mustache;
+      extension = "svg";
+    };
+    kvantumPackage = pkgs.runCommandLocal "base16-kvantum" {} ''
+      mkdir -p $out/share/Kvantum/Base16Kvantum
+      cat ${kvconfig} >>$out/share/Kvantum/Base16Kvantum/Base16Kvantum.kvconfig
+      cat ${svg} >>$out/share/Kvantum/Base16Kvantum/Base16Kvantum.svg
+    '';
```

Maybe we should extract the common base directory as follows:

```suggestion
    kvantumPackage = let
      directory = "${builtins.placeholder "out"}/share/Kvantum/Base16Kvantum";
    in
      pkgs.runCommandLocal "base16-kvantum" {} ''
        mkdir --parents "${directory}"

        cp --dereference "${kvconfig}" "${directory}/Base16Kvantum.kvconfig"
        cp --dereference "${svg}" "${directory}/Base16Kvantum.svg"
      '';
```

Can someone verify whether the use of the `builtins.placeholder` still requires the `mkdir` statement?

---

This may be simpler to write as a Bash variable:

```suggestion
    kvantumPackage = pkgs.runCommandLocal "base16-kvantum" {} ''
      directory="$out/share/Kvantum/Base16Kvantum"
      mkdir -p "$directory"
      cat ${kvconfig} >>"$directory/Base16Kvantum.kvconfig"
      cat ${svg} >>"$directory/Base16Kvantum.svg"
    '';
```

This way also reduces duplication in the generated script.